src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values() [index]);
```

Comment:
```suggestion
        return String.valueOf(Colors.values()[index]);
```

Reply:
Can`t understand why: "return null;"  
We need return color value? (Colors.values()[index])